common/src/IO/FreeImageTextureReader.cpp

```diff
@@ -57,6 +59,21 @@ namespace TrenchBroom {
             }
         }
 
+        static Color getAverageColor(const Assets::TextureBuffer& buffer, const GLenum format) {
+            ensure(format == GL_RGBA || format == GL_BGRA, "expected RGBA or BGRA");
+
+            const unsigned char* const data = buffer.data();
+
+            Color average;
+            for (int64_t i = 0; i < kdl::col_size<int64_t>(buffer); i += 4) {
```

Comment:
Why not `std::size_t` for the index? The buffer is a `std::vector`, should we not index that with `std::size_t`?